demos/python_demos/text_spotting_demo/text_spotting_demo.py

```diff
@@ -79,7 +79,7 @@ def build_argparser():
                       required=True, type=str, metavar='"<path>"')
     args.add_argument('-i',
                       dest='input_source',
-                      help='Required. Path to an image, video file or a numeric camera ID.',
+                      help='Required. Input to process.',
```

Comment:
```suggestion
                      help='Required. Path to video file or numeric camera ID.',
```
is that correct statement?

Reply:
My change is suggested by https://github.com/opencv/open_model_zoo/blob/develop/CONTRIBUTING.md#demo